test/ReCaptchaTest.php

```diff
@@ -163,4 +157,47 @@ public function testUsesReCaptchaHelper()
         $captcha = new ReCaptcha;
         $this->assertEquals('captcha/recaptcha', $captcha->getHelperName());
     }
+
+    public function testValidationForDifferentElementName()
+    {
+        $captcha = new ReCaptcha([
+            'site_key' => getenv('TESTS_ZEND_SERVICE_RECAPTCHA_SITE_KEY'),
```

Comment:
Agreed with @akrabat , though I've noted a problem with the "skip" message in `setUp()`.